internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -354,6 +359,33 @@ type networkLoadBalancerConfig struct {
 	appDNSName           *string
 }
 
+func convertELBAccessLogsConfig(mft *manifest.Environment) (*template.ELBAccessLogs, error) {
+	elbAccessLogsArgs, isELBAccessLogsSet := mft.ELBAccessLogs()
+	if !isELBAccessLogsSet { //if ELB access logs configs are not defined.
+		return nil, nil
+	}
+
+	var bucketName string
+	bucketPrefix := defaultBucketPrefix
+
+	if elbAccessLogsArgs == nil { // if default ELB access logs has been defined using access_logs: true.
+		return &template.ELBAccessLogs{
+			BucketPrefix: bucketPrefix,
+		}, nil
+	}
+	if elbAccessLogsArgs.BucketName != nil {
+		bucketName = aws.StringValue(elbAccessLogsArgs.BucketName)
+	}
+	if elbAccessLogsArgs.BucketPrefix != nil {
+		bucketPrefix = aws.StringValue(elbAccessLogsArgs.BucketPrefix)
+	}
+
+	return &template.ELBAccessLogs{
+		BucketName:   bucketName,
```

Comment:
nit: can be simplified to
```suggestion
	if elbAccessLogsArgs.BucketPrefix != nil {
		bucketPrefix = aws.StringValue(elbAccessLogsArgs.BucketPrefix)
	}
	return &template.ELBAccessLogs{
		BucketName:   aws.StringValue(elbAccessLogsArgs.BucketName),
```